tools/api_proto_plugin/plugin.py

```diff
@@ -66,7 +66,8 @@ def plugin(output_descriptors):
             f = response.file.add()
             f.name = file_proto.name + od.output_suffix
             # Don't run API proto plugins on things like WKT types etc.
-            if not file_proto.package.startswith('envoy.'):
+            if (not file_proto.package.startswith('envoy.') and
```

Comment:
i wont block on it as there are many examples elsewhere, but i personally would not add a multiline condition like this as its v unreadable preferably i do this...

```python

some_aggregate_condition = (
    thing_i_care_about
   and other_thing_i_care_about)

if some_aggregate_condition:
    ....

```